tests/functional/ctu_failure/test_files/lib.c

```diff
@@ -0,0 +1,5 @@
+int foo(int i)
```

Comment:
We have a separate directory for the [test_projects](https://github.com/Ericsson/codechecker/tree/09837984d59daecb1f799290a06eb9a105c6020b/tests/projects) which will be analyzed during the tests.

Reply:
Ok, I moved it under the projects dir.